doc/compatibility_matrix.rst

```diff
@@ -13,3 +13,9 @@ The following table demonstrates the compatibility between the nRF Connect SDK a
 +-----------------------------------------+-------------------------+-------------------------------------------------+-----------------------------+
 | v1.14.4                                 | v2.4.0                  | v1.14                                           | Qualified/approved          |
 +-----------------------------------------+-------------------------+-------------------------------------------------+-----------------------------+
+| v1.14.5-dev1                            | v2.4.1                  | v1.14                                           | Qualified/approved          |
++-----------------------------------------+-------------------------+-------------------------------------------------+-----------------------------+
+| v2.5.0                                  | v2.5.0                  | v1.14                                           | Qualified/approved          |
++-----------------------------------------+-------------------------+-------------------------------------------------+-----------------------------+
+
+.. TBC: Should this be updated? And since 14.4 is the first official release shouldn't 14.3 be removed, and 14.5-dev1 added instead?

```

Comment:
@ktaborowski  as per comment from line 21 - should all this information be included?

Reply:
I think v1.14.5-dev1 wasn't qualified, as it was a development tag.
Regarding v2.5.0 I don't know what is current verification status, but as far as I know it wasn't approved yet